gdnative-derive/src/native_script/property_args.rs

```diff
@@ -1,12 +1,20 @@
 pub struct PropertyAttrArgs {
     pub path: Option<String>,
     pub default: Option<syn::Lit>,
+    pub get_prehook: Option<String>,
+    pub get_posthook: Option<String>,
+    pub set_prehook: Option<String>,
+    pub set_posthook: Option<String>,
```

Comment:
Perhaps `Option<syn::Path>` here because it's better to verify that these are actually paths that can be called? I'm also not very sure about the naming. I'd prefer `before_*` and `after_*`, since they're easier to understand than `_prehook` and `_posthook`.

Reply:
Verifying that's it's a path seems fine. How would you feel about taking it one step further and generating and Option<ExprCall> for each one that is the actual function call?

Reply:
I'd prefer separating option parsing and code generation, so it's cleaner to change the implementation later if necessary.

Reply:
I am just backing off on that. The path idea seems pretty good. I'll see what I can do with it.